### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by arunkumarchacko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ To add a dependency on one of the connectors using Maven, use the following:
     <dependency>
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>gcs-connector</artifactId>
-      <version>hadoop2-2.2.2</version>
+      <version>hadoop3-3.0.0</version>
```
Comment (medb) [inline README.md:67]: We do not have `hadoop3-` prefix for 3.0+ releases

**Comment by RustedBones** [comment]
Should we expect a 3.0.0 release soon ? It would unblock us on #864

**Comment by RustedBones** [comment]
Thanks! BTW, the commit message/PR title is confusing. it is probably hadoop 3, release 3.0.0, not 3.3.0, right ?

**Pull request closed without merging**